pkg/sfu/helpers.go

```diff
@@ -55,3 +55,12 @@ func getRttMs(report *rtcp.ReceptionReport) uint32 {
 	ntpDiff := now - report.LastSenderReport - report.Delay
 	return uint32(math.Ceil(float64(ntpDiff) * 1000.0 / 65536.0))
 }
+
+func getCodecNameFromMime(mime string) string {
+	codecName := ""
+	codecParsed := strings.Split(strings.ToLower(mime), "/")
+	if len(codecParsed) > 1 {
+		codecName = codecParsed[1]
```

Comment:
Not sure if this is case sensitive in rtcscore-go. Maybe convert it into lower or upper case to avoid any mismatches or maybe rtcscore-go converts it in which case this is fine.

Reply:
@boks1971 in line 61, its converted to lower case before Split. rtcscore-go also converts to lower case